scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/dao/ScheduleDaoImpl.java

```diff
@@ -17,11 +18,17 @@ public ScheduleDaoImpl() {
 		super(ScheduleEntity.class);
 	}
 
+	@SuppressWarnings("unchecked")
```

Comment:
For neater code, to avoid warnings. Narrowing it down as belowis fine or should get rid of it completely?
try {
			@SuppressWarnings("unchecked")
			List<ScheduleEntity> scheduleEntities = entityManager.createNamedQuery(ScheduleEntity.query_schedulesByAppId)
				.setParameter("appId", appId).getResultList();
		return scheduleEntities;